inc/config.php

```diff
@@ -139,7 +139,7 @@
     Toolbox::setDebugMode();
 
     if (isset($_SESSION["glpiroot"]) && $CFG_GLPI["root_doc"] != $_SESSION["glpiroot"]) {
```

Comment:
I had a issues testing this feature because of this check.

In my case, when trying to load `http://localhost:8083`, I've got:
* `$CFG_GLPI["root_doc"]` -> empty
* `$_SESSION["glpiroot"]` ->glpi/10.0-bugfixes (which was my previous root doc that GLPI apparently kept in my session)

So the condition is true and I'm redirected to an invalid URL (`http://localhost:8083/glpi/10.0-bugfixes`).
If I comment the condition, GLPI work as expected.

After commenting the code, logging out then restoring the code everything seems to work correctly.

To conclude :
We either need a way to detect and clear this value or indicate to administrators that they need to purge users sessions when enabling this feature.

Reply:
Fixed by #14251.